glossary/es/index.md

```diff
@@ -1,98 +1,6 @@
 ---
-redirect_from: /es/glossary.html
-section: glossary
+redirect_from: /en/glossary.html
+layout: glossary
```

Comment:
@morchickit could you give some guides / advices on what or how to get this page done? At first I followed what is written on the [Translating Guide](http://opendatahandbook.org/contribute/translate-glossary/), but then I found some information is missing related with the main file and how to get the page done … I've tried to follow what @jgkim did on the pages he worked on … but it seems was not enough …